docs/topics/configmap-mission-basic-interaction-vertx.adoc

```diff
@@ -19,34 +19,11 @@ $ oc edit configmap app-config
 Change the value for the `message` key to `Bonjour, %s from a ConfigMap !` and save the file.
 
 
-. Update of the config map should be read by the application within an acceptable time (a few seconds) without requiring a restart of the application.
-. Use `curl` to execute a `GET` request against the booster with the updated ConfigMap configuration. You can also use a browser to do this.
+. Update of the ConfigMap should be read by the application within an acceptable time (a few seconds) without requiring a restart of the application.
```

Comment:
Question: We have the `:name-mission-configmap:` attribute, which evaluates to _ConfigMap._ Would the usage of this attribute in this case be warranted, or do we want to hardcode it like it is?

Reply:
Nice catch:) IMHO, I'd be more inclined to use a variable there. Will fix.